src/components/ServerErrorView.jsx

```diff
@@ -30,25 +30,23 @@ const MainContent = styled(Box)(({ theme }) => ({
 }));
 
 function ServerErrorView({ status }) {
-  let systemsCount = 0;
   let systemRows = [];
   if (status.apiIsUp) {
     const { systems } = status.serverStatus;
-    systemsCount = Object.keys(systems).length;
-    systemRows = Object.keys(status.serverStatus.systems).map((member) => ({
+    systemRows = Object.keys(systems).map((member) => ({
       system: member,
-      system_is_up: status.serverStatus.systems[member].ok,
+      system_is_up: systems[member].ok,
     }));
   }
 
   const columns = [
     {
       label: 'System',
-      property: 'system',
+      name: 'system',
```

Comment:
This is the required fix. I did look a converting this file to typescript (which would've caught this error) but it wasn't obvious to me how to do it.